ManagerDTO.cs

```diff
@@ -0,0 +1,117 @@
+﻿using System;
+using System.Reflection;
+using System.Text;
+using AirportTickets.Booking;
+using AirportTickets.Flight;
+using static AirportTickets.Flight.FlightDTO;
+
+namespace AirportTickets
+{
+    public class ManagerDTO
+    {
+        public string Name { get; set; }
+
+        public ManagerDTO()
+        {
+
+        }
+
+        public List<ReservationDTO> FilterByFlight(FlightDTO flight)
+        {
+            return
+                BookingDTO.RetreiveBookings()
+                .Where(item => item.Flight == flight)
+                .ToList();
+        }
+
+        public string PropertyAttributes()
```

Comment:
`GetFlightValidations` ? We usually use verbs in the method names to understand the action performed in the method.